src/game/components/CCPropsItemWeapon.cpp

```diff
@@ -71,12 +71,23 @@ bool CCPropsItemWeapon::GetPropertyNumPtr(PropertyIndex_t iPropIndex, PropertyVa
     switch (iPropIndex)
     {
         case PROPIWEAP_RANGEL:
+        {
+        	if (_uiRange == 0) {
```

Comment:
Could you please use the code style convention for spaces/tabulations and for brackets?

Reply:
I haven't got what you mean but I think you mean me to use the rule that move brackets new line, if it's I am already using it, but it's like habit for me to remove, but I can move brackets new line if it would be better :)

Reply:
Yup we have bracket style and indentation (use spaces instead of tabs) at the bottom of readme.md!

Reply:
I done the inline changes and changed all my codes with the new inlines.